web/modules/custom/error_page/README.md

```diff
@@ -0,0 +1,132 @@
+core `FinalExceptionSubscriber` by showing the output in its own way and
+stopping the propagation of the `KernelEvents::EXCEPTION` event.
+
+## Fatal and user errors
+
+The module uses its own error/exception handlers, instead of
+`_drupal_error_handler()` and `_drupal_exception_handler()`. Because those
+procedural functions are hard to be extended or overwritten, the core code is
+90% copied in module's handlers. These are slightly changed to allow using a
+custom outputted HTML rendered markup.  
+
+# Getting started
+
+## Install
+
+Install the module as any other module. There are no other dependencies.
+
+## Configure
+
+As explained earlier, the entire configuration is done via `settings/php`:
+
+### ErrorPageErrorHandler autoload
+
+As the container might not be available yet when an uncaught exception or a
+fatal error occur, the auto-loading might not work for extensions such as
+modules. Thus we cannot register our error/exception handlers, which live in
+`ErrorPageErrorHandler`. The system must be instructed explicitly from where to
+load the class. If you are using composer this is done automatically because in
+the module's `composer.json` we've added the class as a `classmap` entry of
+`autoload`. If, for some reasons, you're not using Composer, then you'll need to
+explicitly require the file. In this case, in `settings.php` just add this line:
+
+```php
+// Note that the path to error handler might be different on some installations.
+require_once 'modules/custom/error_page/src/ErrorPageErrorHandler.php';
+```
+
+### Set the custom error handlers
+
+Next two lines should be added in `settings.php`:
+
+```php
+set_error_handler(['Drupal\error_page\ErrorPageErrorHandler', 'handleError']);
+set_exception_handler([
+  'Drupal\error_page\ErrorPageErrorHandler',
+  'handleException',
+]);
+```
+
+### Configuration
+
+In `settings.php`:
+
+```php
+// Defaults to TRUE.
+$settings['error_page']['uuid']['enabled'] = TRUE;
+// Defaults to TRUE.
+$settings['error_page']['uuid']['add_to_message'] = TRUE;
+// Point to the path where the customizable HTML markup files are placed. It's
+// recommended that the custom template location is placed outside the webtree
+// or is protected from the public access with a file, such as markup/.htaccess.
+$settings['error_page']['template_dir'] = DRUPAL_ROOT . '/../path/to/templates';
+```
+
+### All together
+
+Your `settings.php` section might look like:
+
+```php
+// Only if you don't use composer.
+require_once 'modules/custom/error_page/src/ErrorPageErrorHandler.php';
+set_error_handler(['Drupal\error_page\ErrorPageErrorHandler', 'handleError']);
+set_exception_handler([
+  'Drupal\error_page\ErrorPageErrorHandler',
+  'handleException',
+]);
+// Log the UUID in the Drupal logs.
+$settings['error_page']['uuid']['enabled'] = TRUE;
+// Don't inject the UUID into the logged message. Just keep it in variables.
+$settings['error_page']['uuid']['add_to_message'] = FALSE;
```

Comment:
I fail to see the value of this. Why just not add it if available?

Reply:
yes, agree. let's remove it.